pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -230,6 +264,18 @@ async def validate_assembly(self):
                 f"Product mismatch: {group_product} != {self.product}. This pipeline only supports {self.product}."
             )
 
+    async def get_release_date(self):
```

Comment:
From what I know, the release date will be set in the assembly config, and we should rely on that

Reply:
gen-assembly is not currently adding the release date to the assembly config ([example](https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com/job/aos-cd-builds/job/build%252Fgen-assembly/634/console)). So prepare-release can currently only fetch the release date from the release schedule

Reply:
yes, https://github.com/openshift-eng/art-tools/pull/1782 plan to add that, then we can use `group_config.get("release_date", "")`, I mean let's not overlap the work

Reply:
yes, i agree with ximin. Would propose:
- use date if --date is set
- if not rely on group_config.get("release_date", "")
- complain if not found